src/dsverifier.cpp

```diff
@@ -1832,48 +1854,499 @@ void check_minimum_phase_delta_domain()
 }
 
 /*******************************************************************
- Function: check_state_space_stability
+ Function: y_k
 
- Inputs:
+ Inputs: A, B, C, D, u, k, x0
```

Comment:
What is the meaning of `A, B, C, D, u, k, x0`?

Reply:
The meaning is in the function definition.